README.md

```diff
@@ -12,9 +12,9 @@ This is [Shareable Config Presets](https://docs.renovatebot.com/config-presets/)
 ## Usage
 
 <!-- markdownlint-disable-next-line no-inline-html -->
-<details><summary><u>If Renovate has already been activated for repo</u></summary>
+<details><summary>If Renovate has already been activated for repo:</summary>
```

Comment:
A colon has a tendency to mislead users

```suggestion
<details><summary><u>If Renovate has already been activated for repo</u></summary>
```

Reply:
The underline doesn't render on github, which is why I removed it. I'm indifferent towards the colon though 😄

Reply:
Ah, Okay, interesting, why then in MD ext in VS Code with Github style setting it renders 🤔 
Nice catch!